common-crafting.lic

```diff
@@ -197,6 +197,51 @@ module DRCC
     true
   end
 
+  def get_adjust_tongs(usage, bag, bag_items, belt, adjustable_tongs)
+    
+    DRCC.get_crafting_item('tongs', bag, bag_items, belt) unless DRCI.in_hands?('tongs')
+    return false unless adjustable_tongs
+
+    case usage
+    when 'shovel'
+      if @tongs_status == 'shovel'
```

Comment:
here, return false unless adjustable_tongs
get tongs

Reply:
though probably unnecessary, if they're not adjustable, tongs_status wouldn't be defined.